src/display/editor/ink.js

```diff
@@ -372,30 +383,18 @@ class InkEditor extends AnnotationEditor {
       this.opacity ??= InkEditor._defaultOpacity;
     }
     this.currentPath.push([x, y]);
-    this.#lastPoint = null;
+    this.#hasSomethingToDraw = false;
     this.#setStroke();
-    this.ctx.beginPath();
-    this.ctx.moveTo(x, y);
 
     this.#requestFrameCallback = () => {
-      if (!this.#requestFrameCallback) {
-        return;
+      if (this.#hasSomethingToDraw) {
+        this.#drawPoints();
+        this.#hasSomethingToDraw = false;
       }
```

Comment:
Would it perhaps be safer to call `this.#drawPoints();` *unconditionally* here, and instead add the following check at the top of that method?
```javascript
    #drawPoints() {
      if (!this.#hasSomethingToDraw) {
        return;
      }
      this.#hasSomethingToDraw = false;
```